scripts/fetch-external-resources

```diff
@@ -17,18 +18,27 @@ RESOURCES = [
     {
       :layout => 'post',
       :title => 'Changelog',
-    }
+    },
+    nil
   ],
   [
     'https://updates.jenkins.io/latestCore.txt',
     'content/_tmp/latestCore.txt',
+    nil,
     nil
   ],
   [
     'https://updates.jenkins.io/stable/latestCore.txt',
     'content/_tmp/latestLTSCore.txt',
+    nil,
     nil
   ],
+  [
+    'https://jenkins.ci.cloudbees.com/job/infra/job/PipelineStepsDoc/lastSuccessfulBuild/artifact/allAscii.zip',
```

Comment:
I don't have the ability to add a job to jenkins.io and wanted to get this piece into the website.  The pipeline job on jenkins.ci can be moved onto ci.jenkins.io as long as the pipeline-utility-steps plugin is installed (needed for `zip`).